gateway_code/open_nodes/tests/open_nodes_test.py

```diff
@@ -18,37 +18,30 @@
 #
 # The fact that you are presently reading this means that you have had
 # knowledge of the CeCILL license and that you accept its terms.
-"""open_nodes package tests."""
+""" open nodes plugins tests """
 
+from __future__ import print_function
 import unittest
-import mock
 
-from .. import node_class, all_nodes_types
-from ..node_m3 import NodeM3
-from ..node_a8 import NodeA8
+from gateway_code.nodes import open_node_class, all_open_nodes_types
+from gateway_code.open_nodes.node_m3 import M3
```

Comment:
Why not calling them iotlab_m3 and iotlab_a8 ?

Reply:
True, that might make it clear that these are the reference implementations making it clear the difference with custom nodes

Reply:
I'll keep the node type attribute TYPE to `'m3'` and `'a8'` though, it's that way in install lib and other places

Reply:
It's gonna be IotlabM3 and IotlabA8 classes should be CamelCase